HangFire.Autofac/HangFire.Autofac.csproj

```diff
@@ -1,92 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
The same. Consider migrating `HangFire.Autofac` to new csproj format instead of using xproj and project.json.